bindings/java/src/main/com/apple/foundationdb/FDB.java

```diff
@@ -37,7 +37,7 @@
  *   This call is required before using any other part of the API. The call allows
  *   an error to be thrown at this point to prevent client code from accessing a later library
  *   with incorrect assumptions from the current version. The latest supported API version
- *   is defined as {@link ApiVersion.LATEST}.<br><br>
+ *   is defined as ApiVersion.LATEST.<br><br>
```

Comment:
This is actually technically incorrect, but I also can't seem to find a way to make it work. `create_javadoc` has been handed both this file and the generated `APIVersion` file, but doesn't seem to connect the two for some reason?